Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```diff
@@ -79,3 +79,56 @@
   - type: Appearance
     visuals:
     - type: BarrelBoltVisualizer
+
+- type: entity
+  name: Flintlock pistol
+  parent: SniperBase
+  id: FlintlockPistol
+  description: A pirate's companion, Yarrr!
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Snipers/flintlock.rsi
+  - type: RangedWeapon
+  - type: BoltActionBarrel
+    caliber: AntiMaterial
+    capacity: 1
+    fireRate: 1.0
+    minAngle: 25
+    maxAngle: 80
+    angleIncrease: 20
+    angleDecay: 15
+    fillPrototype: CartridgeAntiMaterial
+    soundInsert:
+      path: /Audio/Weapons/Guns/MagIn/bullet_insert.ogg
+  - type: Appearance
+    visuals:
+    - type: BarrelBoltVisualizer
+  - type: Item
+    icon: Objects/Weapons/Guns/Snipers/flintlock.rsi
+
+
+- type: entity
+  name: Musket
+  parent: SniperBase
+  id: Musket
+  description: This should've been in a museum long before you were born.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Snipers/musket.rsi
+  - type: RangedWeapon
+  - type: BoltActionBarrel
+    caliber: AntiMaterial
```

Comment:
i dont know if this makes any sense for muskets but Based

Reply:
I just thought making the muskets high damage would make them more unique, and while on paper it seems really powerful to have a 60 damage rifle/pistol I believe they are balanced by their inaccuracy, and the hassle of reloading after one shot in the midst of combat. Then again, I don't know much about balance so maybe it should be changed, it's up to you I guess.